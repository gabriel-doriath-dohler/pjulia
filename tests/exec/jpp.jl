function useless(n::Int64, s::String)
	for i =1:n
		println(s)
	end
end

useless(42, "The Answer to the Ultimate Question of Life, The Universe, and Everything.")

