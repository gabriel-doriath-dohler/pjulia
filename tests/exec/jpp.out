The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
The Answer to the Ultimate Question of Life, The Universe, and Everything.
